op-acceptance-tests/justfile

```diff
@@ -0,0 +1,25 @@
+# Default recipe - runs the acceptance tests with default parameters
+default:
+    @just acceptance-test
+
+# Run acceptance tests against a devnet
+acceptance-test devnet="simple" gate="localnet":
+    #!/usr/bin/env bash
+    set -euo pipefail
+
+    # First run the appropriate devnet from the kurtosis-devnet directory if needed
+    # we ignore failures here because the devnet may already be running
+    (cd ../kurtosis-devnet && just {{ devnet }}-devnet) || true
```

Comment:
This should do:

`just ../kurtosis-devnet/{{ devnet }}-devnet`

Also that *should* be idempotent, so we shouldn't need to ignore errors.

Reply:
Very nice simplification - nice one and thanks!

As for ignoring errors; I am finding it's needed after the first execution. I get an error to the effect that the devnet already exists:
`Caused by: Error response from daemon: network with name kt-simple-devnet already exists`
So I figured we could put this `||true` here, at least temporarily.

Reply:
ok, let's remember to investigate this